docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -255,6 +255,92 @@ Standard TensorFlow\* operations:
 | ZerosLike | No |
 
 
+## TensorFlow 2 Keras\* Supported Operations
+
+Standard TensorFlow 2 Keras\* operations:
+
+| Operation Name in TensorFlow 2 Keras\* | Limitations|
```

Comment:
@popovaan, @evolosen, could you please double-check each layer (you tested) whether I have missed something in limitations?